tests/tests/swfs/visual/fonts/device-font/test.toml

```diff
@@ -0,0 +1,19 @@
+num_frames = 1
+
+[fonts.regular]
+family = "Tinos"
```

Comment:
We could use any other font, maybe with an even more permissive license. I was just looking at Tinos anyway.